authentication/http.go

```diff
@@ -59,7 +60,17 @@ func WithAccessToken() Middleware {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			rawToken := r.Header.Get("Authorization")
-			token := rawToken[strings.LastIndex(rawToken, " ")+1:]
+			token := strings.TrimPrefix(rawToken, "Bearer ")
```

Comment:
This is a change to the behavior in the code. It seemed safer than the original code, if `Bearer token` is the only variant of `Authorization` header that is supported.

The new code is more strict in that it only removes a pre-defined prefix. The old version would accept a header in the form `Bearer firsttoken secondtoken` and read `secondtoken` from the header, which might lead to exploits if another component only evaluates the `firsttoken`.